Neos.Flow/Tests/Unit/Mvc/Routing/RouterTest.php

```diff
@@ -99,101 +102,13 @@ protected function setUp(): void
         $this->mockActionRequest = $this->getMockBuilder(ActionRequest::class)->disableOriginalConstructor()->getMock();
     }
 
-    /**
-     * @test
-     */
-    public function resolveCallsCreateRoutesFromConfiguration()
```

Comment:
we should carefully review if not too many testcases were removed here and if they were copied to the routing provider tests

Reply:
The tests were either obsolete, testing nothing, or moved to ConfigurationProviderTest (new) or RoutesTest also new.